### PR TITLE
Added Cosmos DB for MongoDB vCore

### PR DESCRIPTION
**Pull request by khelanmodi**
Adding @jcodella as a contributor

### CODE REVIEW

**Comment by rharige** [comment]
We have an entry for Azure AI Search, as well as MongoDB in the table. Since some of the links provided point to Azure AI Search docs - can you clarify how is this new entry different?

**Comment by dhruv-anand-aintech** [comment]
> We have an entry for Azure AI Search, as well as MongoDB in the table. Since some of the links provided point to Azure AI Search docs - can you clarify how is this new entry different?

@rharige I believe this is a separate DB product supporting Vector Search with MongoDB as core, and Azure AI Search is a separate Vector DB.

**Comment by dhruv-anand-aintech** [comment]
@khelanmodi @jcodella Please add references to documentation or code examples for each functionality in the source_url field. Thanks!

File: docs/tools/vdb_table/data/azurecosmosdbmongodbvcore.json [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+      "support": "none",
+      "source_url": "",
+      "comment": ""
+    },
+    "license": {
+      "value": "Proprietary",
+      "source_url": "",
+      "comment": ""
+    },
+    "dev_languages": {
```
Comment (msmygit) [inline docs/tools/vdb_table/data/azurecosmosdbmongodbvcore.json:21]: This is the development language of the database itself. It was advised to me to only have 1 or 2 maximum primary development languages for this part earlier and you may want to update it as such. TY!

**Comment by svonava** [comment]
@khelanmodi are you still planning to get this in, or should we close the PR for now?

**Comment by svonava** [comment]
@khelanmodi @msmygit going to close this if we can't address @dhruv-anand-aintech's comment above

**Pull request closed without merging**